### PR TITLE
Apply Swagger CSS and HTML to Collector/Search Job API

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request adds  Swagger CSS and HTML style to the Collector Mgmt and Search Job APIs, which are not part of our API site/yml file. Facilitates API use for customers.

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/api/collector-management/collector-api-methods-examples.md [hunk 1]
```diff
@@ -372,16 +387,20 @@ curl -u '<accessId>:<accessKey>' -X GET https://api.sumologic.com/api/v1/collect
...
+
 ## POST methods
 
 ### Create Hosted Collector
 
+<details>
+<summary><span className="api post">POST</span><code>/collectors</code></summary>
+<p/>
+
 Use the POST method with a JSON file to create a new Hosted Collector. The required parameters can be referenced in the [Response fields](#Response_fields) table above. Note that "id" field should be omitted when creating a new Hosted Collector.
```
Comment (jpipkin1) [inline docs/api/collector-management/collector-api-methods-examples.md:400]:
```suggestion
Use the POST method with a JSON file to create a new Hosted Collector. The required parameters can be referenced in the [Response fields](#response-fields) table above. Note that "id" field should be omitted when creating a new Hosted Collector.
```

File: docs/api/collector-management/collector-api-methods-examples.md [hunk 2]
```diff
@@ -427,15 +446,20 @@ curl -u '<accessId>:<accessKey>' -X POST -H "Content-Type: application/json" -T
...
+
 ## PUT methods  
 
 ### Update a Collector  
 
+<details>
+<summary><span className="api put">PUT</span><code>/collectors/&#123;id&#125;</code></summary>
+<p/>
+
 Use the PUT method with your JSON file to update an existing Collector. Available parameters can be referenced in the [Response fields](#Response-fields) table above. The JSON request file must specify values for all required fields. Not modifiable fields must match their current values in the system. This is in accordance with HTTP 1.1 RFC-2616 Section 9.6.
```
Comment (jpipkin1) [inline docs/api/collector-management/collector-api-methods-examples.md:459]:
```suggestion
Use the PUT method with your JSON file to update an existing Collector. Available parameters can be referenced in the [Response fields](#response-fields) table above. The JSON request file must specify values for all required fields. Not modifiable fields must match their current values in the system. This is in accordance with HTTP 1.1 RFC-2616 Section 9.6.
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @kimsauce. I had only a couple of suggestions.

**Comment by kimsauce** [comment]
Great catches! thank you, @jpipkin1 .

**Pull request merged**